tests/testDisplay.py

```diff
@@ -90,6 +90,36 @@ def testWith(self):
         with afwDisplay.getDisplay(0) as disp:
             self.assertIsNotNone(disp)
 
+    def testDefaultBackend(self):
+        """Test setting and getting of default backend"""
+        if (backend != 'virtualDevice'):
```

Comment:
Yes, the new tests will not be run by Jenkins.  A proper test using `testDisplay.py` requires specifying a backend (`ds9` or `firefly` currently) and at least one user click to test everything. Since the virtual backend takes no optional arguments, it is not possible to test the defaultBackend setting of these with it.